### PR TITLE
Quick Fix and Pragma for check 103

### PR DESCRIPTION
**Pull request by fidley**
#1134

### CODE REVIEW

**Comment by larshp** [comment]
fixed by adding exemption

**Comment by fidley** [comment]
Need someone to check on older releases ;)

File: src/checks/zcl_aoc_check_103.clas.locals_imp.abap [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+DEFINE add_replacement_proposal.
+  CLEAR lv_proposal.
+  lv_proposal-from = &1.
+  lv_proposal-sequence = &2.
+  lv_proposal-to = &3.
+  lv_proposal-oss_note = &4.
+  INSERT lv_proposal INTO TABLE rt_proposals.
+END-OF-DEFINITION.
+
+CLASS lcl_quickfix DEFINITION FINAL.
```
Comment (larshp) [inline src/checks/zcl_aoc_check_103.clas.locals_imp.abap:14]: perhaps move this class to a global class, so more checks can use it? assuming it works

Comment (fidley) [reply src/checks/zcl_aoc_check_103.clas.locals_imp.abap:14]: This was the idea for the next step, if it works for more persons. 
Then I'd move this maybe to the abstract class or to super class, at least those parts which can be reused. I'd also put some small documentation for it.

**Comment by fidley** [comment]
image

image

image

File: src/checks/zcl_aoc_check_103.clas.locals_imp.abap (hunk 1 above)
Comment (larshp) [reply src/checks/zcl_aoc_check_103.clas.locals_imp.abap:14]: composition over inheritance

**Comment by fidley** [comment]
Mass processing with recommended QF
image

**Pull request merged**